Examples/Google/Compute/COMPILE.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Maybe a silly question, but do you expect generation to work on Windows?

Reply:
Windows-literacy is definitely a weakness of mine.

Reply:
We don't have to do it right now, but just something to bring up.  I care more that the final product is usable for windows devs, which we can verify with a kokoro windows test.

Reply:
Actually Swift support on Windows is pretty nascent. https://swiftforwindows.github.io/